dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SwaggerClientIT.java

```diff
@@ -590,6 +590,9 @@ public void testStarStarredTool() throws ApiException, IOException, TimeoutExcep
         assertEquals(2, containerId);
         StarRequest request = SwaggerUtility.createStarRequest(true);
         containersApi.starEntry(containerId, request);
+        List<User> starredUsers = containersApi.getStarredUsers(container.getId());
+        Assert.assertEquals(1, starredUsers.size());
+        starredUsers.forEach(user -> Assert.assertNull("User profile is not lazy loaded in starred users", user.getUserProfile()));
```

Comment:
question: is the user profile is not lazy loaded for starring requests, are avatars being displayed properly for the stargazers page?